src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -105,4 +105,103 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->by_primary_key( 'id', $attendee->ID )
+			              ->set_args( $attendee_data );
+		}
+		catch ( Tribe__Repository__Usage_Error $e ) {
+			do_action( 'tribe_log', 'error', __CLASS__, [
+				'message' => $e->getMessage(),
+			] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: cb6c04cfb69731543002b0f896dac60c</sub>